### DISCUSSION TITLE
How do I run a test experiment?

### DISCUSSION THREAD

**Discussion by elanhickler** [RobinSchmidt/RS-MET#315] [Q&A] [answered]
I'm trying to run the ResamplingExperiments, specifically timeWarp() to see if the warp map / resampling stuff is working because I can't get it to work in my project.

**Comment by RobinSchmidt** [comment, marked as answer]
open the solution: 

TestsRosicAndRapt.sln

in

RS-MET\Tests\TestsRosicAndRapt\Builds\VisualStudio2019

and then in Main.cpp uncomment one or more lines for the experiment(s), you want to run (and comment any experiment that you don't want to run but is currently uncommented)

**Comment by RobinSchmidt** [comment]
i added a file to the repo:

https://github.com/RobinSchmidt/RS-MET/blob/master/Workflows.txt

this might be convenient for me as well - because i tend to forget my workflows as well, when i haven't used them in a while. ...apparently, i even consider this important enough to put it the top-level directory :-O
